atomicapp/providers/openshift.py

```diff
@@ -123,7 +125,7 @@ class OpenShiftProvider(Provider):
     kubernetes_api = None
     access_token = None
     namespace = DEFAULT_NAMESPACE
-    ssl_verify = False
+    ssl_verify = True
```

Comment:
I think we should add a comment to this one to explain that it is mapped to the `verify` argument of various python-requests function calls and that it can take on the value of True || False || path to ca, etc..